ledger/ledger-on-memory/BUILD.bazel

```diff
@@ -135,37 +135,6 @@ da_scala_binary(
     ],
 )
 
-# Full conformance test (single node setup)
-conformance_test(
-    name = "conformance-test",
-    ports = [6865],
-    server = ":app",
-    server_args = [
-        "--contract-id-seeding=testing-weak",
-        "--participant=participant-id=example,port=6865",
-    ],
-    test_tool_args = [
-        "--verbose",
-    ],
-)
```

Comment:
I think it's worth keeping this one; it actually tests that _ledger-on-memory_ works, which is important until we decide to remove it.

Reply:
It testing a subset of the functionality of the multi-participant and split-participant tests. I would suggest to let it go. This entire project is on the way out and we are leaving the only 2 tests that test otherwise untested corners.